services/server/src/fawry/lib/signature.ts

```diff
@@ -9,12 +10,21 @@ import {
 } from "@/fawry/types/requests";
 
 function generateSignature(...data: unknown[]) {
```

Comment:
Using the `unkown` type, give the assumption that it can accept any thing but in reality it only accepts strings or numbers.


```suggestion
function generateSignature(...data: Array<string | number>) {
```